WordPress/build.gradle

```
@@ -455,12 +465,41 @@ task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.
     ]
 }
 
-task printVersionName {
+task printWordPressVersionName {
+    doLast {
+        println android.productFlavors.wordpress.versionName-rc-1
+    }
+}
+
+task printWordPressVersionCode {
     doLast {
-        println android.productFlavors.vanilla.versionName
+        println 1
     }
 }
 
+task printJetpackVersionName {
+    doLast {
+        println android.productFlavors.jetpack.versionName
+    }
+}
+
+task printJetpackVersionCode {
+    doLast {
+        println android.productFlavors.jetpack.versionCode
+    }
+}
+
+
+task "updateVersionProperties" {
```

It looks like this task is still a work in progress since it doesn't really do anything at the moment. It can be quite tricky to get the version number increased automatically. We could look into using [nebula plugin](https://nebula-plugins.github.io/) or at least use it as an inspiration.

Also, I don't mind this being a part of the project right now, but it'd be good to have this as a Gradle plugin so that we can use it in other apps as well.

---

The idea here for this task is just to update the properties file but the Nebula plugin looks a bit neater, I'm checking it out and their approach now.

---

```suggestion
tasks.register("updateVersionProperties") {
```

https://docs.gradle.org/current/userguide/task_configuration_avoidance.html